cache/redisOps.js

```diff
@@ -171,7 +171,8 @@ function deleteKeys(type, objectName, name) {
       const subjectKey = nameParts[0];
       const aspect = nameParts[1];
       if ((objectName.toLowerCase() === subjectType && nameKey === subjectKey)
-        || (objectName.toLowerCase() === aspectType && name === aspect)) {
+        || (objectName.toLowerCase() === aspectType &&
+          name.toLowerCase() === aspect)) {
```

Comment:
name may be camelCase, but aspect will be lowercase. 
If the case don't match, samples won't be deleted.

Reply:
name here will always be lower case. the name is the key to be deleted and the toKey in "/cache/sampleStore.js" function already converts it to lower case

Reply:
although the key will be lowecase, the name is can be camelCase as it is passed as a param, from the db/model/aspect.js